os/pm/pm_activity.c

```diff
@@ -130,6 +133,25 @@ void pm_activity(int domain, int priority)
 
 		pdom->accum = (int16_t)accum;
 
+#ifdef CONFIG_PM_METRICS
+		struct pm_accumchange_s *node = NULL;
+		
+		if (name == NULL) {
+			pmdbg("ERROR: name is null\n");
+		} else {
+			for (node = (struct pm_accumchange_s*)sq_peek(&(pdom->each_accum)); node; node = (struct pm_accumchange_s*)(node->entry.flink)) {
+				if (!strncmp(node->name, name, strlen(name) + 1)) {
```

Comment:
For strxxx APIs, please add `<string.h>`.

Reply:
modified